app/controllers/display_ad_events_controller.rb

```diff
@@ -1,5 +1,7 @@
-class DisplayAdEventsController < ApplicationController
+class DisplayAdEventsController < ApplicationMetalController
+  include ActionController::Head
```

Comment:
we might want to move this `include ActionController::Head` directly in the `ApplicationMetalController`

Reply:
Even though we use it in all the current examples, I could see future `ApplicationMetalController` instances including something else instead of `Head`.

Also I think the explicit inclusion here sticks with the spirit of the functionality and acts as a bit of a demo for future readers of this code.